external/src/main/java/com/redhat/cloud/policies/engine/LightweightEngineImpl.java

```diff
@@ -257,6 +260,18 @@ private Set<Alert> fireTriggers(Event event) {
         }
     }
 
+    private boolean shouldSendNotification(FullTrigger fullTrigger) {
+        if (fullTrigger.getTrigger().getActions() != null) {
+            for (TriggerAction triggerAction : fullTrigger.getTrigger().getActions()) {
+                if (triggerAction.getActionPlugin().equals("email") || triggerAction.getActionPlugin().equals("notification")) {
```

Comment:
@josejulio I'm not about checking `notification`, it might not be necessary. WDYT?

Reply:
This looks good - Wondering now if we verify if the trigger/policy is enabled before processing it - i don't recall seeing that but unsure if it was somewhere else outside of the LightweightEngine.

Reply:
Good question. I already checked with the logs that triggers are (re)loaded in `LightweightEngineImpl` when enabled. I need to check that they are removed when disabled.

Reply:
Something's fishy with disabled triggers. I'll investigate that a bit later.